glass/fields.py

```diff
@@ -29,25 +29,24 @@
 import warnings
 
 # typing
-from typing import Any, Callable, Generator, Iterable, Optional, Sequence, Tuple, Union
+from typing import Any, Callable, Generator, Iterable, Sequence
 
 import healpy as hp
 import numpy as np
 import numpy.typing as npt
 from gaussiancl import gaussiancl
 
 # types
-Array = np.ndarray
-Size = Union[None, int, Tuple[int, ...]]
-Iternorm = Tuple[Optional[int], Array, Array]
-ClTransform = Union[str, Callable[[Array], Array]]
-Cls = Sequence[Union[Array, Sequence[float]]]
-Alms = np.ndarray
+Size = None | int | tuple[int, ...]
+Iternorm = tuple[int | None, npt.NDArray, npt.NDArray]
+ClTransform = str | Callable[[npt.NDArray], npt.NDArray]
+Cls = Sequence[npt.NDArray | Sequence[float]]
+Alms = npt.NDArray
```

Comment:
I have tried this before and the newer syntax does not work when you are making your own custom types.

Reply:
Well let's see. I got an error, but that looked to be from it being `None | <x>` rather `<x> | None`.

Reply:
Yeah, it was that. Have restored `Optional`, `Tuple`, `Union` here